python/docs/verification.mdx

```diff
@@ -25,7 +25,7 @@ Console messages logged in the page can be brought into the Playwright context.
 
 ```py
 # Listen for all console logs
-page.on("console", msg => print(msg.text))
```

Comment:
This should be fixed in the Playwright repo (https://github.com/microsoft/playwright/blob/master/docs/src/verification.md) and then [rolled](https://github.com/microsoft/playwright.dev#roll-docs) to playwright.dev Otherwise it will be overwritten with the next roll.